CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # CHANGELOG
 
 ## Not released
+
+- Client Side Widgets [#62](https://github.com/CartoDB/carto-react-lib/pull/62)
```

Comment:
Why is this line added in this PR? I don't see anything related to CSW here.

Reply:
It was missed in last PR.

Reply:
I rather having a separate PR for this kind of small mistakes that can be merged quickly to keep everything clear and tidy, and avoid losing this item in changelog in case of this PR is not merged before releasing a new version. But if we are sure this PR is going to be merged soon before releasing, it can stay for this time.

Reply:
Not sure if it will be merged soon.. So let's remove it for the moment, and will create a small PR for that :+1: